Libs/MRML/Core/vtkMRMLAbstractViewNode.h

```diff
@@ -298,6 +299,18 @@ class VTK_MRML_EXPORT vtkMRMLAbstractViewNode : public vtkMRMLNode
   vtkSetMacro(ScreenScaleFactor, double);
   //@}
 
+  //@{
+  /// \brief Get/Set the displayable manager group associated with the view.
+  vtkGetObjectMacro(DisplayableManagerGroup, vtkObject);
+  vtkSetObjectMacro(DisplayableManagerGroup, vtkObject);
```

Comment:
At the minimum, the `Set` functions should check the type using `IsA()`[^1] ..

```
  if (obj && !obj->IsA("vtkMRMLDisplayableManagerGroup"))
    {
    vtkErrorWithObjectMacro(this, "SetDisplayableManagerGroup: expected vtkMRMLDisplayableManagerGroup");
    return;
    }
```

[^1]: https://vtk.org/doc/nightly/html/classvtkObjectBase.html#a44ecbcd4243dc6d6cbf619c9f983a438